modules/http4s-client/src/main/scala/io/janstenpickle/trace4cats/http4s/client/Http4sClientRequest.scala

```diff
@@ -0,0 +1,17 @@
+package io.janstenpickle.trace4cats.http4s.client
+
+import io.janstenpickle.trace4cats.http4s.common.Request_
+import io.janstenpickle.trace4cats.model.AttributeValue.{LongValue, StringValue}
+import io.janstenpickle.trace4cats.model.{AttributeValue, SemanticAttributeKeys}
+import org.http4s.Uri
+
+object Http4sClientRequest {
+  def toAttributes(req: Request_): Map[String, AttributeValue] =
+    req.uri.host.map {
+      case address @ Uri.Ipv4Address(_, _, _, _) =>
```

Comment:
why not `case address: Uri.Ipv4Address => `?

Reply:
Because I'm using an extractor for `host` below, and it's not good practice to mix different pattern matching styles.